File: src/main/java/org/goafabric/core/data/controller/vo/Address.java
package org.goafabric.core.data.controller.vo;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Null;

public record Address (
        String id,
        @Null String version,

        @NotNull String use,
        @NotNull String street,
        @NotNull String city,
        @NotNull String postalCode,
        @NotNull String state,
        @NotNull String country
) {}

File: src/main/kotlin/org/goafabric/core/organization/controller/dto/Address.kt
package org.goafabric.core.organization.controller.dto

data class Address(
    val id: String? = null,
    val version: Long? = null,
    val use: String? = null,
    val street: String? = null,
    val city: String? = null,
    val postalCode: String? = null,
    val state: String? = null,
    val country: String? = null
)
